docs/storage-abstraction.md

```diff
@@ -361,45 +285,48 @@ Set<String> namespaces = admin.getNamespaceNames();
 You can get the tables of a namespace as follows:
 
 ```java
-// Get the tables of the namespace "ns"
+// Get the tables of the namespace "ns".
 Set<String> tables = admin.getNamespaceTableNames("ns");
 ```
 
-#### Get a table metadata
+### Get table metadata
```

Comment:
@Torch3333 Same as with the `### Get existing namespaces` section, this section should only exist in the `master` branch, right? If so, I will remove this section when we backport it to `3.10` and earlier branch versions.